client/js/App.vue

```diff
@@ -18,7 +18,7 @@ export default {
 </script>
 
 <style lang="scss">
-@import url('https://fonts.googleapis.com/css2?family=PT+Serif:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Infant:wght@400;600;700&family=Libre+Baskerville:wght@400;700&family=PT+Serif:wght@400;700&display=swap');
```

Comment:
Good job combining these, would love to see if we need all the weights for each of these fonts. A future improvement could be to only load them when we need them (gameview), but that's grasping at straws for our usages and a higher level of complexity for a minor improvement.

Reply:
Good call out, currently we are only using one weight for each font. Not sure if that would change in the future.